spec/gitlab_custom_hooks.rb

```diff
@@ -0,0 +1,87 @@
+require 'support/publicize_methods.rb'
+require 'spec_helper'
+require 'gitlab_custom_hook'
+
+describe GitlabCustomHook do
+  let(:repository_path) { "/home/git/repositories" }
+  let(:repo_name)   { 'dzaporozhets/gitlab-ci' }
+  let(:repo_path)  { File.join(repository_path, repo_name) + ".git" }
+  let(:gitlab_custom_hooks) { GitlabCustomHook.new }
+  let(:local_pre_receive) {File.join(repo_path,'custom_hook','pre-receive')}
+  let(:local_post_receive) {File.join(repo_path,'custom_hook','post-receive')}
+  let(:global_pre_receive) {File.join(ROOT_PATH,'custom_hook','pre-receive')}
+  let(:global_post_receive) {File.join(ROOT_PATH,'custom_hook','post-receive')}
+
+  describe "select hook file" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.